src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/MassTransitClientExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using MassTransit;
+using MassTransit.Monitoring;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ;
+
+/// <summary>
+/// MassTransitClientExtensions provides extension methods for configuring MassTransit in a client application.
+/// </summary>
+public static class MassTransitClientExtensions
+{
+    /// <summary>
+    /// Configures MassTransit with RabbitMQ integration for the client side,
+    /// using the same configuration used by the hosting environment.
+    /// </summary>
+    /// <param name="services">The client service collection.</param>
+    /// <param name="name">A unique name for the RabbitMQ instance.</param>
+    /// <param name="telemetry">Enables telemetry, which could be exported to either OpenTelemetry or Application Insights.</param>
+    public static void AddMassTransitClient(this IServiceCollection services, string name, bool? telemetry = false)
```

Comment:
Client integrations should be extensions of `IHostApplicationBuilder` not `IServiceCollection`